pom.xml

```diff
@@ -21,7 +21,7 @@
 
   <groupId>io.cdap.plugins</groupId>
   <artifactId>elasticsearch-plugins</artifactId>
-  <version>1.10.0-SNAPSHOT</version>
+  <version>1.10.1-SNAPSHOT</version>
```

Comment:
No need to change the snapshot version since it is not released yet.

Reply:
Understood. Fixed this as well.